src/chromeDebug.ts

```diff
@@ -14,9 +16,10 @@ const EXTENSION_NAME = 'debugger-for-chrome';
 declare let VERSION: string;
 let versionWithDefault = typeof VERSION === 'undefined' ? 'unspecified' : VERSION; // Not built with webpack for tests
 
-// non-.txt file types can't be uploaded to github
-// also note that __dirname here is out/
-const logFilePath = path.resolve(__dirname, '../vscode-chrome-debug.txt');
+// non-.txt file types can't be uploaded to github. Generated filename is something like: vscode-chrome-debug_2017-07-19_161210.txt
+const timestampForFilename = new Date().toISOString().replace(/T/, '_').replace(/:/g, '').replace(/\..*/, '');
```

Comment:
No, then it would just get huge. I just prefer overwriting one file but we can keep this if it's what you need.

Technically we could specify this in the launch config, because the file name isn't actually used until the logger is initialized. It's just bad design on my part to set up the hypothetical file name ahead of time. But I don't care enough to change it right now.